tests/openwisp2/settings.py

```diff
@@ -162,6 +162,11 @@
     CELERY_TASK_EAGER_PROPAGATES = True
     CELERY_BROKER_URL = 'memory://'
 
+# Workaround for stalled migrate command
+CELERY_BROKER_TRANSPORT_OPTIONS = {
+    'max_retries': 10,
```

Comment:
Patch, looks good, my only question is how much time would it take for the `10` retries?
Or in other words, how long would the user wait before seeing the error?

Reply:
For 10 tries, it will take around 1.5 minutes. If it sounds a lot, we can just lower this number. I forgot to mention that we will have to add this in the test project of openwisp-controller as well.